packages/trpc/server/routers/viewer/organizations/update.handler.ts

```diff
@@ -20,11 +21,13 @@ type UpdateOptions = {
 
 export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
   // A user can only have one org so we pass in their currentOrgId here
-  const currentOrgId = ctx.user?.organization?.id;
+  const currentOrgId = ctx.user?.organization?.id || input.id;
```

Comment:
Needed to pass down an org Id instead of inferring from user orgId as an admin doesn't have an org assigned.